stripe-ui-core/src/test/java/com/stripe/android/uicore/elements/DropdownScreenshotTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.stripe.android.uicore.elements
+
+import com.stripe.android.uicore.utils.PaparazziRule
+import com.stripe.android.uicore.utils.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+
+class DropdownScreenshotTest {
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        SystemAppearance.values()
+    )
```

Comment:
😍

Reply:
Nice! 

Since we're starting with screenshot tests for UI components in the ui core module, we might want to introduce [Showkase](https://github.com/airbnb/Showkase) as well, so that we get screenshot testing out of composable @Previews without having to write these.